politeiawww/convert.go

```diff
@@ -142,10 +142,11 @@ func convertCastVotesFromDecredplugin(cv []decredplugin.CastVote) []www.CastVote
 	return cvr
 }
 
-func convertVoteResultsReplyFromDecredplugin(vrr decredplugin.VoteResultsReply) www.VoteResultsReply {
+func convertVoteResultsReplyFromDecredplugin(vrr decredplugin.VoteResultsReply, ir inventoryRecord) www.VoteResultsReply {
```

Comment:
There needs to be a conversion the other way around too.

Reply:
Where will we need to convert from `www.VoteResulsReply` to `decredPlugin.VoteResultsReply`?